lib/models/builder.js

```diff
@@ -84,6 +86,9 @@ module.exports = Task.extend({
       process.on('SIGINT',  this.onSIGINT.bind(this));
       process.on('SIGTERM', this.onSIGTERM.bind(this));
       process.on('message', this.onMessage.bind(this));
+      exit.onExit(function(code) {
+        return this.cleanupAndExit(code);
+      }.bind(this));
```

Comment:
I think this could be simplified to `exit.onExit(this.cleanup.bind(this));` and we could then probably remove `cleanupAndExit()`